gravitee-am-service/src/test/java/io/gravitee/am/service/CertificateServiceTest.java

```diff
@@ -50,6 +66,40 @@
 @RunWith(MockitoJUnitRunner.class)
 public class CertificateServiceTest {
 
+    public static final String DOMAIN_NAME = "my-domain";
+    public static final String CERTIFICATE_SCHEMA_DEFINITION = "{\n" +
```

Comment:
I think an actual json file kept in src/test/resources would be more maintainable than this.